sys/linux/bpf_386.const

```diff
@@ -140,7 +140,7 @@ BPF_PROG_TYPE_CGROUP_SOCK = 9
 BPF_PROG_TYPE_CGROUP_SOCKOPT = 25
 BPF_PROG_TYPE_CGROUP_SOCK_ADDR = 18
 BPF_PROG_TYPE_CGROUP_SYSCTL = 23
-BPF_PROG_TYPE_EXT = 28
+# BPF_PROG_TYPE_EXT is not set
```

Comment:
There is a number of such changes in *.const files. You probably used an older linux-next revision. Please fetch linux-next and re-execute 'make extract'.